### DISCUSSION TITLE
How do I distribute a Trilium notebook?

### DISCUSSION THREAD

**Discussion by Calab** [TriliumNext/Trilium#4688] [Q&A] [answered]
I've created a small notebook in Trilium. I would like to share it with another Trilium user. How can I send the notebook to them?

I know you can export a notebook to a ZIP file containing HTML documents but  I don't see how I would use that to share the notebook to another Trilium user.

Is it not possible to have multiple Trilium notebooks on one computer?

**Comment by meichthys** [comment]
You can use the "Share" feature to make a public link of your notebook that anyone can view using that link.
https://github.com/zadam/trilium/wiki/Sharing

**Reply by Calab** [reply]
Unfortunately a server based solution won't work. We need to access the notebook when offline.

**Reply by meichthys** [reply]
Ok, Then the manual export/import should work for you as mentioned here: https://github.com/zadam/trilium/discussions/4688#discussioncomment-8776872

**Comment by sigaloid** [comment, marked as answer]
You can export that zip file then simply import it, which will load that subtree.

**Reply by Calab** [reply]
Thank you. That is what I needed to know.

As a side note, I think it would be better to use a different extension for the Trilium files instead of just zip. That way the extension could be associated to the Trilium program and opened directly. I know that I normally would not want to merge someone else's notes with my own notebook.

**Reply by Calab** [reply]
@sigaloid ... is there any way to manage multiple notebooks on one system?  I don't see any way to do a "File... Open" to open a different notebook.

Having a user import someone else's notes into their own notebook will get messy very quickly. There are also times that you CANNOT too much related data in one location, for security reasons.

I find that being able to manage a ".tlm" file is much better than some system database that I cannot handle directly.

**Reply by sigaloid** [reply]
There's a single database located in the data directory (read the wiki) that would act as what you want, but there's no way to manage multiple notebooks. The way to do it is handle subtrees and share the exported zip files.

**Reply by rom1dep** [reply]
@Calab You can probably run multiples instances, each using their own Document, with this: https://github.com/zadam/trilium/wiki/Data-directory#changing-the-location-of-data-directory
